### PR TITLE
语句优化

### PR DESCRIPTION
**Pull request by ichiyouda**
3. …… if we had ignored calls,  感觉这句译为「如果我们有被忽略的调用」，读起来更好理解一些，希望不是错译。

**目标章节**：1-js/06-advanced-functions/09-call-apply-decorators/04-throttle

**当前上游最新 commit**：https://github.com/javascript-tutorial/en.javascript.info/commit/ec84c3f8d8adad98f20b45ff4fb1a02b3722677e

**本 PR 所做更改如下：**

文件名 | 参考上游 commit | 更改（理由）
-|-|-
solution.md | a23882d | 无

### CODE REVIEW

File: 1-js/06-advanced-functions/09-call-apply-decorators/04-throttle/solution.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ function throttle(func, ms) {
 调用 `throttle(func, ms)` 返回 `wrapper`。
 
 1. 在第一次调用期间，`wrapper` 只运行 `func` 并设置冷却状态（`isThrottled = true`）。
-2. 在这种状态下，所有调用都记忆在 `savedArgs/savedThis` 中。请注意，上下文和参数（arguments）同等重要，应该被记下来。我们同时需要他们以重现调用。
-3. ……然后经过 `ms` 毫秒后，触发 `setTimeout`。冷却状态被移除（`isThrottled = false`），如果我们忽略了调用，则将使用最后记忆的参数和上下文执行 `wrapper`。
+2. 在冷却状态下，所有调用都被保存在 `savedArgs/savedThis` 中。请注意，上下文（this）和参数（arguments）都很重要，应该被保存下来。我们同时需要它们来重现调用。
+3. 经过 `ms` 毫秒后，`setTimeout`中的函数被触发。冷却状态被移除（`isThrottled = false`），如果我们有被忽略（return）的调用，将使用最后保存的参数和上下文运行 `wrapper`。
```
Comment (songhn233) [inline 1-js/06-advanced-functions/09-call-apply-decorators/04-throttle/solution.md:36]:
```suggestion
3. 经过 `ms` 毫秒后，`setTimeout`中的函数被触发。冷却状态被移除（`isThrottled = false`），如果存在被忽略的调用，将使用最后一次调用保存的参数和上下文运行 `wrapper`。
```

File: 1-js/06-advanced-functions/09-call-apply-decorators/04-throttle/solution.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ function throttle(func, ms) {
...
 
-第 3 步运行的不是 `func`，而是 `wrapper`，因为我们不仅需要执行 `func`，还需要再次进入冷却状态并设置 timeout 以重置它。
+第 3 步运行的不是 `func`，而是 `wrapper`，因为我们不仅需要执行 `func`，还需要再次进入冷却状态并设置延时以重置节流。
```
Comment (songhn233) [inline 1-js/06-advanced-functions/09-call-apply-decorators/04-throttle/solution.md:38]:
```suggestion
第 3 步运行的不是 `func`，而是 `wrapper`，因为我们不仅需要执行 `func`，还需要再次进入冷却状态并设置 `setTimeout` 以重置节流。
```

**Review by songhn233 — CHANGES_REQUESTED** [review]

**Comment by songhn233** [comment]
Hey @ichiyouda. Thanks for your contribution. I've left some minor  changes to fix, other than that the PR looks good to me!

File: 1-js/06-advanced-functions/09-call-apply-decorators/04-throttle/solution.md (hunk 1 above)
Comment (ichiyouda) [reply 1-js/06-advanced-functions/09-call-apply-decorators/04-throttle/solution.md:36]: 哇！读起来舒服多了

**Comment by ichiyouda** [comment]
> 1. Please make the requested changes.  
> ……
> 2. Add more commits by pushing to the master branch on ichiyouda/zh.javascript.info. 

呜呜，对 Github 不太熟悉。改 PR 我只需要按照第二条的提示做就 Okay 了吧 😌

File: 1-js/06-advanced-functions/09-call-apply-decorators/04-throttle/solution.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ function throttle(func, ms) {
 调用 `throttle(func, ms)` 返回 `wrapper`。
 
 1. 在第一次调用期间，`wrapper` 只运行 `func` 并设置冷却状态（`isThrottled = true`）。
-2. 在这种状态下，所有调用都记忆在 `savedArgs/savedThis` 中。请注意，上下文和参数（arguments）同等重要，应该被记下来。我们同时需要他们以重现调用。
-3. ……然后经过 `ms` 毫秒后，触发 `setTimeout`。冷却状态被移除（`isThrottled = false`），如果我们忽略了调用，则将使用最后记忆的参数和上下文执行 `wrapper`。
+2. 在冷却状态下，所有调用都被保存在 `savedArgs/savedThis` 中。请注意，上下文（this）和参数（arguments）都很重要，应该被保存下来。我们同时需要它们来重现调用。
```
Comment (leviding) [inline 1-js/06-advanced-functions/09-call-apply-decorators/04-throttle/solution.md:35]:
```suggestion
2. 在冷却状态下，所有调用都被保存在 `savedArgs/savedThis` 中。请注意，上下文（this）和参数（arguments）都很重要，应该被保存下来。我们需要它们来重现调用。
```

File: 1-js/06-advanced-functions/09-call-apply-decorators/04-throttle/solution.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ function throttle(func, ms) {
 调用 `throttle(func, ms)` 返回 `wrapper`。
 
 1. 在第一次调用期间，`wrapper` 只运行 `func` 并设置冷却状态（`isThrottled = true`）。
-2. 在这种状态下，所有调用都记忆在 `savedArgs/savedThis` 中。请注意，上下文和参数（arguments）同等重要，应该被记下来。我们同时需要他们以重现调用。
-3. ……然后经过 `ms` 毫秒后，触发 `setTimeout`。冷却状态被移除（`isThrottled = false`），如果我们忽略了调用，则将使用最后记忆的参数和上下文执行 `wrapper`。
+2. 在冷却状态下，所有调用都被保存在 `savedArgs/savedThis` 中。请注意，上下文（this）和参数（arguments）都很重要，应该被保存下来。我们需要它们来重现调用。
+3. 经过 `ms` 毫秒后，`setTimeout`中的函数被触发。冷却状态被移除（`isThrottled = false`），如果存在被忽略的调用，将使用最后保存的参数和上下文运行 `wrapper`。
```
Comment (leviding) [inline 1-js/06-advanced-functions/09-call-apply-decorators/04-throttle/solution.md:36]:
```suggestion
3. 经过 `ms` 毫秒后，`setTimeout`中的函数被触发。冷却状态被移除（`isThrottled = false`），如果存在被忽略的调用，将使用最后一次调用保存的参数和上下文运行 `wrapper`。
```

File: 1-js/06-advanced-functions/09-call-apply-decorators/04-throttle/solution.md [hunk 5]
```diff
@@ -32,7 +32,7 @@ function throttle(func, ms) {
...
 
-第 3 步运行的不是 `func`，而是 `wrapper`，因为我们不仅需要执行 `func`，还需要再次进入冷却状态并设置 timeout 以重置它。
+第 3 步运行的不是 `func`，而是 `wrapper`，因为我们不仅需要运行 `func`，还需要再次进入冷却状态并设置`setTimeout`以重置节流。
```
Comment (leviding) [inline 1-js/06-advanced-functions/09-call-apply-decorators/04-throttle/solution.md:38]:
```suggestion
第 3 步运行的不是 `func`，而是 `wrapper`，因为我们不仅需要执行 `func`，还需要再次进入冷却状态并设置 `setTimeout` 以重置节流。
```

**Review by leviding — APPROVED** [review]

**Comment by leviding** [comment]
Thanks all!

**Pull request merged**